src/main/scala/beam/agentsim/infrastructure/SplitParkingManager.scala

```diff
@@ -0,0 +1,269 @@
+package beam.agentsim.infrastructure
+
+import java.util.concurrent.TimeUnit
+
+import akka.actor.{ActorLogging, ActorRef, Props}
+import akka.util.Timeout
+import beam.agentsim.Resource.ReleaseParkingStall
+import beam.agentsim.infrastructure.SplitParkingManager.{ParkingCluster, Worker}
```

Comment:
allow to select ParkingManager from config, e.g. existing one is called DefaultParkingManager

rename SplitParkingManager -> HierarchicalParkingManager

Reply:
done:
beam.agentsim.taz.parkingManager.managerType = "hierarchical" | "default"
beam.agentsim.taz.parkingManager.hierarchical.numberOfClusters = 8